src/target/compilation_config.cc

```diff
@@ -61,47 +61,47 @@ void CompilationConfigNode::EstablishDefaultSEScopes(const transform::PassContex
   if (host_target.defined()) {
     CHECK(!host_target->host.defined()) << "Host targets are not expected to have hosts";
     host_device_type = static_cast<DLDeviceType>(host_target->kind->device_type);
-    DLOG(INFO) << "Using the given host target " << host_target->ToDebugString()
-               << " of device type " << host_device_type << " for the host target";
+    VLOG(1) << "Using the given host target " << host_target->ToDebugString() << " of device type "
+            << host_device_type << " for the host target";
```

Comment:
Have we discussed VLOG vs DLOG? It seems like people are still adding DLOGs. Might be worth discussing unifying around one debugging system at some point in the future.

Reply:
Yeah. In this case these were my own DLOGs and they were too verbose even for me.

In general getting folks to think about debugability when writing code in the first place would be a fine thing. Eg I don't know if there's any way to see TE, or Schedules.